tests/pages/component/donation-goals-activation.js

```diff
@@ -0,0 +1,12 @@
+import {
+  attribute,
+  clickable
+} from 'ember-cli-page-object';
+
+export default({
+  scope: '.donationGoalsActivation',
+
+  clickActivateDonationGoals: clickable('.activate-donations'),
```

Comment:
A reminder we want to move away from class selectors, so ideally this is `clickable('[data-test-activate-donations]')` and you modify the hbs to add the `data-test-` selector.